sql/CMakeLists.txt

```diff
@@ -214,18 +214,27 @@ ELSE()
   SET(MYSQLD_SOURCE main.cc ${DTRACE_PROBES_ALL})
 ENDIF()
 
-IF(MSVC)
+IF(MSVC OR CMAKE_SYSTEM_NAME MATCHES AIX)
   SET(libs_to_export_symbols sql mysys dbug strings)
   # Create shared library of already compiled object
   # Export all symbols from selected libraries, to be used
   # by plugins
-  ADD_LIBRARY(server SHARED
-    $<TARGET_OBJECTS:sql>
-    $<TARGET_OBJECTS:mysys>
-    $<TARGET_OBJECTS:dbug>
-    $<TARGET_OBJECTS:strings>
-    ${PROJECT_BINARY_DIR}/versioninfo_dll.rc
-  )
+  IF(MSVC)
```

Comment:
That diff is unnecessarily large, and is prone to breakage on AIX, whenever anything new comes in addition to sql,mysys,dbg and strings

``` 
IF(MSVC)
  SET(VERSIONINFO_RC ${PROJECT_BINARY_DIR}/versioninfo_dll.rc)
ELSE()
  SET(VERSIONINFO_RC)
ENDIF()

ADD_LIBRARY(server SHARED
      $<TARGET_OBJECTS:sql>
      $<TARGET_OBJECTS:mysys>
      $<TARGET_OBJECTS:dbug>
      $<TARGET_OBJECTS:strings>
      ${VERSIONINFO_RC}
)

Reply:
Done.